packages/@aws-cdk/aws-route53/test/hosted-zone.test.ts

```diff
@@ -228,4 +228,32 @@ describe('Vpc', () => {
       Name: Match.anyValue(),
     });
   });
-});
+
+  describe('PrivateHostedZone', () => {
+    test('fromPrivateHostedZoneAttributes resource import', () => {
```

Comment:
I'll have a more into the testing part, would be nice to get some feedback if I'm going in the right direction